manage_user_update.php

```diff
@@ -177,20 +177,25 @@
 if( $f_send_email_notification ) {
 	lang_push( user_pref_get_language( $f_user_id ) );
 	$t_changes = '';
+
 	if( strcmp( $f_username, $t_old_username ) ) {
-		$t_changes .= lang_get( 'username_label' ) . lang_get( 'word_separator' ) . $t_old_username . ' => ' . $f_username . "\n";
+		$t_changes .= lang_get( 'username_label' ) . ' ' . $t_old_username . ' => ' . $f_username . "\n";
```

Comment:
Not sure what's the right solution for it, we had hardcoded ': ' in master-1.2.x
https://github.com/mantisbt/mantisbt/blob/master-1.2.x/manage_user_update.php#L152
Furthermore we had just language string username but not username and username_label (which are the same at the moment in English)
In some languages there is the ':' in language string $s_username_label but not in all languages.
The same applies to other strings (e.g. realname, email)